[CrowdStrikeFalcon] new function: "safe_strptime" (#41384)
demisto/content: Packs/Base/ReleaseNotes/1_41_25.md

#### Scripts

##### CommonServerPython

> - Fixed an issue where the CommonServerPython script would throw an error when parsing timestamps without microsecond components.

Reviewer: ```suggestion
- Fixed an issue with the timestamp parsing logic to prevent errors when microsecond components are missing.
```
